dynawo/sources/Common/test/Test.cpp

```diff
@@ -430,7 +430,7 @@ TEST(CommonTest, testSparseMatrix) {
     smj4.addTerm(i, i);
   }
   for (int icol = 0; icol < smj4.nbCol(); ++icol) {
-    for (int ind = smj4.Ap_[icol]; ind < smj4.Ap_[icol + 1]; ++ind) {
+    for (int ind = smj4.Ap_[icol]; ind < static_cast<int>(smj4.Ap_[icol + 1]); ++ind) {
```

Comment:
Wouldn't it be better to replace the int index with an unsigned?

Reply:
OK